content/en/docs/tasks/traffic-management/ingress/gateway-api/index.md

```diff
@@ -10,29 +10,48 @@ owner: istio/wg-networking-maintainers
+This task describes how to configure Istio to expose a service outside the service mesh cluster, using the Kubernetes [Gateway API](https://gateway-api.sigs.k8s.io/).
 These APIs are an actively developed evolution of the Kubernetes [Service](https://kubernetes.io/docs/concepts/services-networking/service/)
 and [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) APIs.
 
+{{< warning >}}
+This feature is currently considered [alpha](/docs/releases/feature-stages/).
+Both the API (owned by Kubernetes SIG-NETWORK) and the Istio implementation are likely to change before being promoted further.
+{{< /warning >}}
+
 ## Setup
 
-1. Install the Gateway API CRDs:
+1. The Gateway APIs do not come installed by default on most Kubernetes clusters. Install the Gateway API CRDs if they are not present:
 
     {{< text bash >}}
-    $ kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.4.0" | kubectl apply -f -
+    $ kubectl get crd gateways.gateway.networking.k8s.io || { kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.4.0" | kubectl apply -f -; }
     {{< /text >}}
 
-1. Install Istio:
+## Differences from Istio APIs
 
-    {{< text bash >}}
-    $ istioctl install
-    {{< /text >}}
+The Gateway APIs share a lot of similarities to the Istio APIs (such as Gateway and VirtualService).
+The main resource even shares the same name (`Gateway`), and the resources serve similar goals.
+
+The new Gateway APIs aim to take the learnings from various Kubernetes ingress implementations, including Istio,
+to build a standardized vendor neutral API. These APIs generally serve the same purposes as Istio Gateway and VirtualService,
+with a few key differences:
+
+* In Istio APIs, a `Gateway` *configures* an existing gateway Deployment/Service that has [been deployed](/docs/setup/additional-setup/gateway/).
+  In the Gateway APIs, the `Gateway` resource both configures *and* deploys a gateway.
+  See [Deployment Methods](#deployment-methods) for more information.
+* In the Istio `VirtualService`, all protocols are configured within a single resource.
+  In the Gateway APIs, each protocol type has its own resource, such as `HTTPRoute` and `TCPRoute`.
+* While the Gateway APIs offer a lot of rich routing functionality, it does not yet cover 100% of Istio's feature set.
+  Work is ongoing to extend the API to cover these use cases, as well as utilizing the APIs [extensibility](https://gateway-api.sigs.k8s.io/#gateway-api-concepts)
+  to better expose Istio functionality.
 
 ## Configuring a Gateway
 
 See the [Gateway API](https://gateway-api.sigs.k8s.io/) documentation for information about the APIs.
 
-1. Deploy a test application:
+In this example, we will deploy a simple application and expose it externally using a `Gateway`.
+
+1. First, deploy a test application:
 
```

Comment:
In a later step, we mention ingress IP, but use INGRESS_HOST. Not sure if there is a reason to change that.